wiki/compatibility.md

```diff
@@ -0,0 +1,6 @@
+## Compatibility
+
+The version of Eiffel REMReM Semantics should implement the desired version of [Eiffel REMReM Protocol Interface](https://eiffel-community.github.io/eiffel-remrem-protocol-interface/).
```

Comment:
This link should point to the GitHub repository instead of GitHub pages.

Reply:
changed the links to point to github repositories

Reply:
changed the link to point to github repository